tests/pytest/core/test_analytics.py

```diff
@@ -80,3 +80,49 @@ def test_Event_update_user_properties(app_request):
 
     assert key in event.user_properties
     assert event.user_properties[key] == value
+
+
+@pytest.mark.django_db
+def test_ViewedPageEvent_sets_default_event_properties(app_request, mocker):
+    update_spy = mocker.spy(ViewedPageEvent, "update_event_properties")
+
+    ViewedPageEvent(app_request)
+
+    utm_codes = ["utm_campaign", "utm_source", "utm_medium", "utm_content", "utm_id"]
+    for utm_code in utm_codes:
+        assert utm_code in update_spy.call_args.kwargs
+        assert update_spy.call_args.kwargs.get(utm_code) is None
+
+
+@pytest.mark.django_db
+def test_ViewedPageEvent_sets_default_user_properties(app_request, mocker):
+    update_spy = mocker.spy(ViewedPageEvent, "update_user_properties")
+
+    ViewedPageEvent(app_request)
+
+    utm_codes = ["utm_campaign", "utm_source", "utm_medium", "utm_content", "utm_id"]
+    for utm_code in utm_codes:
+        assert utm_code in update_spy.call_args.kwargs
+        assert update_spy.call_args.kwargs.get(utm_code) is None
+
+
+@pytest.mark.django_db
+def test_ViewedPageEvent_update_event_properties(app_request):
+    key, value = "utm_campaign", "Transit"
+    event = ViewedPageEvent(app_request)
+
+    event.update_event_properties(**{key: value})
```

Comment:
This is still a test of the `update_event_properties` method -- remember, what you are testing is simply the `ViewedPageEvent` constructor (with the new behavior you added to call `update_event_properties` etc.) We don't want additional calls to client code (e.g. a function on the `ViewedPageEvent`) because that introduces "another thing" that is under test.

The outline of the test could be something like:

```python
# prepare app_request so that is has UTM codes with values in the query parameters
utm_data = {"utm_campaign", "campaign", "utm_source": "source", etc.}
app_request.something = utm_data

# create the event, this is the only thing under test! No other methods from benefits should be called.
event = ViewedPageEvent(app_request)

# now assert that the event has the expected properties:
for key, value in utm_data:
    assert event.event_properties[key] == value
```

This time you have to set up the `app_request` somehow so that is has UTM information that the `ViewedPageEvent` can read (which is what you are later asserting happened).